src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionTestResultViewController.swift

```diff
@@ -37,7 +37,7 @@ class ExposureSubmissionTestResultViewController: DynamicTableViewController, Fo
 	override func viewDidAppear(_ animated: Bool) {
 		super.viewDidAppear(animated)
 		
-		viewModel.updateWarnOthers()
+		viewModel.evaluateShowing()
```

Comment:
I don't understand the naming. Is this func showing something or not?

Reply:
It evaluates the showing of test (if the test result was shown). We have this function already in coronaTestService and familyMemberCoronaTestService. It just calls those functions.